playbooks/common/openshift-cluster/redeploy-certificates/etcd.yml

```diff
@@ -50,6 +52,7 @@
       etcd_peers: "{{ groups.oo_etcd_to_config | default([], true) }}"
       etcd_certificates_etcd_hosts: "{{ groups.oo_etcd_to_config | default([], true) }}"
       openshift_ca_host: "{{ groups.oo_first_master.0 }}"
+      r_etcd_common_etcd_runtime: "{{ openshift.common.etcd_runtime }}"
```

Comment:
I see where this is headed.  I don't think we should be passing in role variables which will be used by other (dependency) roles.  This is where we need to flatten out our role dependencies first it seems.  The only vars we should be passing in are ones for the role we are calling.